pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -5006,14 +5007,14 @@ status:
 	prt := newPipelineRunTest(t, d)
 	defer prt.Cancel()
 
-	reconciledRun, clients := prt.reconcileRun("foo", prOutOfSync.Name, []string{}, false)
+	reconciledRun, clients := prt.reconcileRun(namespace, prOutOfSyncName, []string{}, false)
 
 	actions := clients.Pipeline.Actions()
 	if len(actions) < 3 {
 		t.Fatalf("Expected client to have at least three action implementation but it has %d", len(actions))
 	}
 
-	_ = getPipelineRunUpdates(t, actions)
+	_ = getTaskRunsForPipelineRun(prt.TestAssets.Ctx, t, clients, namespace, prOutOfSyncName, 2)
```

Comment:
It looks like this is being used just to assert that the correct number of taskruns is associated with the pipelinerun, and that the taskruns themselves are verified later on in the test. Could this be combined somehow instead of ignoring the output of this function? or maybe we only need a helper function for verifying that the number of taskruns associated with the pipelinerun is as expected?